### PR TITLE
[IOTDB-897] Remove duplicate maven plugin in module tsfile and website

### PR DESCRIPTION
**Pull request by yuqi1129**

### CODE REVIEW

File: site/pom.xml [hunk 1]
```diff
@@ -39,30 +39,6 @@
     </distributionManagement>
     <build>
         <plugins>
-            <!-- copy source codes into vue required folders-->
```
Comment (jixuan1989) [inline site/pom.xml:42]: Hi, why you thought this is duplicated...
have you tried `mvn package -pl site -P site` ?

File: hadoop/pom.xml [hunk 2]
```diff
@@ -85,26 +85,6 @@
                     </archive>
                 </configuration>
             </plugin>
-            <plugin>
```
Comment (jixuan1989) [inline hadoop/pom.xml:88]: this plugin is for generating `hadoop-tsfile-0.11.0-SNAPSHOT-jar-with-dependencies.jar` file, which includes all dependencies for using hdfs and tsfile.
If the user uses maven, indeed it is useless, but for users who are not use maven, I think it is useful (or someone who uses maven but there are version conflicts for hadoop dependencies).

So, I have no idea about whether to remove this plugin? either is ok.

What do you think?

File: hadoop/pom.xml [hunk 3]
```diff
@@ -40,7 +40,7 @@
         <dependency>
             <groupId>org.apache.iotdb</groupId>
             <artifactId>tsfile</artifactId>
-            <version>0.11.0-SNAPSHOT</version>
```
Comment (jixuan1989) [inline hadoop/pom.xml:43]: +1

**Review by jixuan1989 — COMMENTED** [review]
have you tried to run on the modified codebase (especially for the site module)?

File: site/pom.xml (hunk 1 above)
Comment (yuqi1129) [reply site/pom.xml:42]: You are right,  i mistake it

File: hadoop/pom.xml (hunk 2 above)
Comment (yuqi1129) [reply hadoop/pom.xml:88]: Ok, if `hadoop-tsfile-0.11.0-SNAPSHOT-jar-with-dependencies.jar` is needed somewhere, i do think there is no enough reason to remove it.

Thanks for your kindly review~

**Comment by yuqi1129** [comment]
This pull pull is mark as invalid, close it

**Pull request closed without merging**